lightning/src/ln/reorg_tests.rs

```diff
@@ -147,7 +147,7 @@ fn do_test_onchain_htlc_reorg(local_commitment: bool, claim: bool) {
 			txdata: vec![],
 		};
 		connect_block(&nodes[1], &block);
-		expect_pending_htlcs_forwardable!(nodes[1]);
+		expect_pending_htlcs_forwardable_and_htlc_handling_failed!(nodes[1], HTLCHandlingFailedConditions::new().htlc_processing_failed_with_reason(HTLCDestination::FailedPayment { payment_hash: our_payment_hash }));
```

Comment:
yeah, I think this may be related to being able to resolve this first https://github.com/lightningdevkit/rust-lightning/pull/1403#discussion_r908520656